### ISSUE TITLE
Camera tab with multicam doesn't set driver mode properly

### ISSUE THREAD

**Issue by mcm001** [PhotonVision/photonvision#221]
https://user-images.githubusercontent.com/29715865/104051316-cd892f80-519c-11eb-91d4-d737d510f6ad.mp4

**Comment by srimanachanta** [comment]
What is the correct behavior? Driver mode persists with camera change?

**Comment by mcm001** [comment]
Yes, the UI should let the first camera go back to what it used to be doing, and make the newly selected camera go into driver mode

**Comment by Gold856** [comment]
Change of opinion. Driver mode should stay with the camera. Closing as WAI.

**Comment by mcm001** [comment]
To be specific, this was OBE by https://github.com/PhotonVision/photonvision/commit/f623e4a1cc147fc736cce2e4634829d6b0945b35
